tests/README.md

```diff
@@ -2,7 +2,7 @@
 
 To run these tests, you need to do the following
 
-1. From a terminal, create a virtual environment at `tests/venv`.
+1. From a terminal, create a virtual environment at `tests/venv`. You need to use `virtualenv` package.
```

Comment:
virtualenv was for Py2. In Py3, it's venv

Reply:
You're right. However, the `run_tests.py` logic expects a virtual env created by `virtualenv` which still can be used on Python 3. The [script expects](https://github.com/shotgunsoftware/tk-3dsmax/blob/master/tests/run_tests.py#L20) the `activate_this.py` file to be part of the virtual environment created by `virtualenv`.